RelVal/assignRelValWorkflow.py

```diff
@@ -31,13 +51,13 @@ def assignRequest(url,workflow,team,site,era,procstr,procver,activity,lfn):
               "MinMergeSize": 2147483648,
               "MaxMergeSize": 4294967296,
               "MaxMergeEvents": 100000,
-              #"maxRSS": 2411724,
-              "maxRSS": 3072000,
-              "maxVSize": 20294967,
+              #"maxRSS": 4911724,
+              "MaxRSS": maxrss,
+              "MaxVSize": 20294967,
```

Comment:
Here @AndrewLevin !
Someone had changed the maxRSS on the main branch but you changed it to use a variable.
So overwrote the old version with yours. Everything else checked fine.